src/project/ProjectModel.js

```diff
@@ -59,6 +59,13 @@ define(function (require, exports, module) {
     var _exclusionListRegEx = /\.pyc$|^\.git$|^\.gitmodules$|^\.svn$|^\.DS_Store$|^Thumbs\.db$|^\.hg$|^CVS$|^\.hgtags$|^\.idea$|^\.c9revisions$|^\.SyncArchive$|^\.SyncID$|^\.SyncIgnore$|\~$/;
 
     /**
+     * @public
+     * Glob definition of files and folders that should be excluded directly
+     * inside node domain watching with chokidar
+     */
+    var exclusionGlob = "**/(.pyc|.git|.gitmodules|.svn|.DS_Store|Thumbs.db|.hg|CVS|.hgtags|.idea|.c9revisions|.SyncArchive|.SyncID|.SyncIgnore|node_modules)";
```

Comment:
I have already written it, but I think this should be an array. 
And chokidar supports string, path and glob.
Could you add `bower_components` too?
Does the `.git` work with your extension? On VSCode i believe they use `.git/objects`